### ISSUE TITLE
Integration test failing regularly with: panic: Error trying to empty bucket

### ISSUE THREAD

**Issue by tleyden** [couchbase/sync_gateway#3218]
I noticed this when trying to validate https://github.com/couchbase/sync_gateway/pull/3217.

http://uberjenkins.sc.couchbase.com/view/Build/job/sync-gateway-integration-master/316/console and other builds on both the feature branch and the master branch are failing with errors like:

```
--- FAIL: TestAdminReduceViewQuery (14.64s)
panic: Error trying to empty bucket.  Spec: {Server:http://integration-cbs.sc.couchbase.com:8091 PoolName: BucketName:test_data_bucket FeedType: Auth:{Username:test_data_bucket Password:password BucketName:test_data_bucket} CouchbaseDriver:GoCBCustomSGTranscoder MaxNumRetries:0 InitialRetrySleepTimeMS:0 UseXattrs:true ViewQueryTimeoutSecs:<nil>}.  err: Timed out waiting for bucket to be empty after flush.  ItemCount: 868 [recovered]
	panic: Error trying to empty bucket.  Spec: {Server:http://integration-cbs.sc.couchbase.com:8091 PoolName: BucketName:test_data_bucket FeedType: Auth:{Username:test_data_bucket Password:password BucketName:test_data_bucket} CouchbaseDriver:GoCBCustomSGTranscoder MaxNumRetries:0 InitialRetrySleepTimeMS:0 UseXattrs:true ViewQueryTimeoutSecs:<nil>}.  err: Timed out waiting for bucket to be empty after flush.  ItemCount: 868
```

**Comment by tleyden** [comment]
This seems to reliably pass with xattrs=false, and reliably fail w/ xattrs=true, and all else equal:

- http://uberjenkins.sc.couchbase.com/view/Build/job/sync-gateway-integration-master/327/parameters/ (fail)
- http://uberjenkins.sc.couchbase.com/view/Build/job/sync-gateway-integration-master/326/parameters/ (pass)

**Comment by bbrks** [comment]
I think this may have been fixed in #3261 

I will leave this issue open for now until there have been more integration tests ran to verify.

**Comment by bbrks** [comment]
xattrs = true now reliably passes. Closing issue.
